### PR TITLE
feat: [CCM-7011]: Business Mapping

### PR DESCRIPTION
**Pull request by jenil-harness**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by jenil-harness** [comment]
retrigger titlecheck

**Review by rajarshee-harness — APPROVED** [review]

**Review by jyotiarora-harness — APPROVED** [review]

**Review by bhavya-sinha — APPROVED** [review]

File: src/modules/75-ce/components/BusinessMappingBuilder/BusinessMappingBuilder.tsx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            })
+          )
+          .required()
+      })
+    )
+  })
+
+  /* istanbul ignore next */
+  const handleSubmit: (values: BusinessMappingForm) => void = async values => {
+    delete values.costTargetsKey
```
Comment (rishik30) [inline src/modules/75-ce/components/BusinessMappingBuilder/BusinessMappingBuilder.tsx:92]: Is it not better to not delete the keys in values but create a new object as per requirements?

File: src/modules/75-ce/components/BusinessMappingBuilder/CostBucketBuilder.tsx [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+                      }}
+                      margin={{
+                        top: 'medium'
+                      }}
+                      padding={{
+                        top: 'medium'
+                      }}
+                    >
+                      <RadioButtonGroup
+                        label="Sharing Strategy"
```
Comment (rishik30) [inline src/modules/75-ce/components/BusinessMappingBuilder/CostBucketBuilder.tsx:255]: string here

File: src/modules/75-ce/components/PerspectiveFilters/PerspectiveFilters.tsx [hunk 3]
```diff
@@ -36,6 +39,11 @@ const PerspectiveFiltersNew: React.FC<PerspectiveFiltersProps> = ({ formikProps
 
   const { data } = result
 
+  const dateRange =
+    (formikProps.values.viewTimeRange?.viewTimeRangeType &&
+      perspectiveDefaultTimeRangeMapper[formikProps.values.viewTimeRange?.viewTimeRangeType]) ||
```
Comment (rishik30) [inline src/modules/75-ce/components/PerspectiveFilters/PerspectiveFilters.tsx:44]: no need to include `?` operator here

File: src/modules/75-ce/components/BusinessMappingBuilder/CostBucketBuilder.tsx (hunk 2 above)
Comment (jenil-harness) [reply src/modules/75-ce/components/BusinessMappingBuilder/CostBucketBuilder.tsx:255]: Will take it up in follow up PR.

File: src/modules/75-ce/components/PerspectiveFilters/PerspectiveFilters.tsx (hunk 3 above)
Comment (jenil-harness) [reply src/modules/75-ce/components/PerspectiveFilters/PerspectiveFilters.tsx:44]: Will take care of it in follow up PR.

File: src/modules/75-ce/components/BusinessMappingBuilder/BusinessMappingBuilder.tsx (hunk 1 above)
Comment (jenil-harness) [reply src/modules/75-ce/components/BusinessMappingBuilder/BusinessMappingBuilder.tsx:92]: 👍 Will take it up in follow up PR.

**Review by vkbansal — APPROVED** [review]

**Pull request merged**